test/inspector/test-inspector.js

```diff
@@ -151,6 +151,37 @@ function testInspectScope(session) {
   ]);
 }
 
+function testCommandLineAPI(session) {
+  console.log('[test]', 'Verify that command line API was injected');
```

Comment:
nit: I'd prefer if the test's did not include unnecessary console.log statements